packages/repository/test/unit/decorator/relation.decorator.unit.ts

```diff
@@ -54,11 +54,6 @@ describe('relation decorator', () => {
         target: () => Address,
       });
 
-      expect(jugglerMeta).to.eql({
```

Comment:
Similarly here - let's rework this check to assert that no property was defined by the relation decorator.

Reply:
The original tests were added and changed accordantly